v2/support/modal_sample_app/assets/iframe.html

```diff
@@ -1,6 +1,6 @@
 <html>
 <head>
-  <link rel="stylesheet" href="https://assets.zendesk.com/apps/sdk-assets/css/2/zendesk_garden.css" type="text/css">
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/combine/npm/@zendeskgarden/css-bedrock@6.0,npm/@zendeskgarden/css-buttons@5.0,npm/@zendeskgarden/css-forms@5.0" type="text/css">
```

Comment:
Can we get these up to latest?
* `css-bedrock@7`
* `css-buttons@6`
* `css-forms@6`